trajectory_msgs/JointTrajectoryPoint traj_point
bool service_called